downstream/assemblies/platform/assembly-migrate-platform.adoc

```diff
@@ -1,10 +1,12 @@
 ifdef::context[:parent-context: {context}]
 
 // [id="assembly-migrate-platform_{context}"]
-= Migrating data to {PlatformNameShort} {PlatformVers}
+= Steps to migrate data to {PlatformNameShort} {PlatformVers}
 
 For platform administrators looking to complete an upgrade to the {PlatformNameShort} {PlatformVers}, there may be additional steps needed to migrate data to a new instance:
 
+To complete an upgrade to {PlatformNameShort} {PlatformVers}, you must migrate your data. Migrating data to a new instance requires additional steps 
```

Comment:
Missing punctuation at the end of this sentence.

Reply:
I fixed the missing punctuation. Please squash and merge when you can. Cheers!